### DISCUSSION TITLE
a-tisket for MusicBrainz Imports

### DISCUSSION THREAD

**Discussion by CurlyBorly** [RandomNinjaAtk/docker-lidarr-extended#52] [General]
I noticed in the [Audio Wiki](https://github.com/RandomNinjaAtk/docker-lidarr-extended/wiki/Extended-Audio-Script-Information) the automated ways to add releases to MusicBrainz from Discogs and Deezer. I didn't attempt the Discogs import but I was unable to get the Deezer script working, and I've read in their Issues that its script is no longer maintained. [a-tisket](https://etc.marlonob.info/atisket/) is a newer alternative that is just a website that you can submit a Deezer/Spotify/iTunes album link and it will fill out all the fields for you. Since its a website, it doesn't rely on using a Firefox extension so it's more accessible, and it has worked well for me so far. Just want to recommend it to be added to the wiki.

**Comment by RandomNinjaAtk** [comment]
Feel free to update the wiki ;)
